play/square-wars/script.js

```diff
@@ -0,0 +1,715 @@
+//global objects for the game
+let player, enemy, enemys, bullet, bullets, boss, gameHandler, mathHandler;
+//global boolean for the game engine
+let gamePlaying;
+
+//difficulty level for game preset to 1
+let difficultyLevel = 1
+
+/**
+* used to set the difficulty level from the html main menu buttons
+* difficultyLevel is used to set up the MathClassHandler
+* @param {String} Level
+**/
+function setLevel(level) {
+  intLevel = parseInt(level)
+  difficultyLevel = intLevel;
+  document.getElementById("gameBtn").style.display = "inline";
+}
+
+/**
+* Fucntion to validate the numbers only in the input text box
+* @param (Event) evt
+**/
+function isNumber(evt) {
+  var iKeyCode = (evt.which) ? evt.which : evt.keyCode
+  if (iKeyCode != 46 && iKeyCode > 31 && (iKeyCode < 48 || iKeyCode > 57)) {
+    return false;
+  }
+  return true;
+ }
+
+
+/**
+  MoveableCanvasObject class is all objects that are moveable on the canavs
+**/
+class MoveableCanvasObject {
+  constructor(width, height, x, y, color) {
+    this.width = width;
+    this.height = height;
+    this.x = x;
+    this.y = y;
+    this.color = color;
+  }
+  update() {
+    let ctx = myGameArea.context;
+    ctx.fillStyle = this.color;
+    ctx.fillRect(this.x, this.y, this.width, this.height);
+  }
+  newPos() {
+    this.x += this.speedX;
+    this.y += this.speedY;
+  }
+
+}
+
+/**
+* player class used used to update play speed and postion on canvas/map clipping
+**/
+class Player extends MoveableCanvasObject {
+  constructor(width, height, x, y, color, killCount, roundCount) {
+    super(width, height, x, y, color);
+    this.speedX = 0;
+    this.speedY = 0;
+    this.killCount = killCount;
+    this.roundCount = roundCount;
+    this.ArrowButtons = false;
```

Comment:
Be consistent with capitalization of variables, for JavaScript camelCase is the standard.